net-mgmt/zabbix-agent/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		zabbix-agent
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Enterprise-class open source distributed monitoring agent
+PLUGIN_DEPENDS=     zabbix32-agent
```

Comment:
is this the actual package that we use? https://github.com/opnsense/tools/blob/master/config/17.1/ports.conf#L88